### DISCUSSION TITLE
加密后脚本无法运行

### DISCUSSION THREAD

**Discussion by yangshuer** [dashingsoft/pyarmor#2020] [Q&A]
命令：
~/JM$ pyarmor g -O dist -r -i config/
~/JM/dist/config$ python load_config.py

错误：
Traceback (most recent call last):
  File "/home/yang/JM/dist/config/load_config.py", line 2, in <module>
    from .pyarmor_runtime_000000 import __pyarmor__
ImportError: attempted relative import with no known parent package

**Comment by jondy** [comment]
搜索文档找解决方案
